### PR TITLE
Release `1.0.0-pre1`

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR updates `base` dependency to `1.0.0-pre1`.

Changes:
- `Identifier.generate(Class) is used where possible.
- Tests updated since now entity IDs are required by default.
- The usages of `Identifier` and `TypeConverter` updates after the [API](https://github.com/SpineEventEngine/base/pull/232) changes.

The framework version was bumped to `1.0.0-pre1`.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: server/src/main/proto/spine/system/server/command_lifecycle.proto [hunk 1]
```diff
@@ -46,7 +46,9 @@ message CommandLifecycle {
     option (entity).kind = AGGREGATE;
 
     // The ID of the command.
-    spine.core.CommandId id = 1;
+    //
+    // The ID is not required since it is set only after the command was received.
```
Comment (armiol) [inline server/src/main/proto/spine/system/server/command_lifecycle.proto]: This is an issue, I think. Let's discuss.

File: server/src/test/proto/spine/test/system/server/q/projections.proto [hunk 2]
```diff
@@ -1,41 +0,0 @@
...
-
-option (type_url_prefix) = "type.spine.io";
-option java_package = "io.spine.system.server";
-option java_outer_classname = "ProjectionsProto";
-option java_multiple_files = true;
-
-import "spine/people/person_name.proto";
-import "spine/test/system/server/entity_history_test.proto";
-
-message PersonDetails {
```
Comment (armiol) [inline server/src/test/proto/spine/test/system/server/q/projections.proto]: I don't understand how this file isn't needed anymore.

Comment (dmytro-grankin) [reply server/src/test/proto/spine/test/system/server/q/projections.proto]: It was just moved to a more appropriate file — `entity_history_test.proto`. I think it is more appropriate to have all definitions for a test suite in a single file while the number of definitions isn't big.

File: server/src/main/proto/spine/system/server/command_lifecycle.proto (hunk 1 above)
Comment (dmytro-grankin) [reply server/src/main/proto/spine/system/server/command_lifecycle.proto]: I have created the issue #877 since it isn't easy to make it required now.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

**Pull request merged**